### ISSUE TITLE
Allow Episode Overrides for Staff Commands

### ISSUE THREAD

**Issue by IanMitchell** [IanMitchell/ARX-7#177]
Hinoe brought up that API support, and bot support, for designating episodes might be helpful (for cases like when FFF had one position delayed on three episodes). An (optional) format might be similar to this:

```
> .done <show> #4
  => Set's <Show> episode 4 staff to done
> .done <show>
  => Set's <Show> lowest unreleased episode staff to done
```

A similar format could be used for Blame and other Showtimes commands.

**Comment by IanMitchell** [comment]
Tagging as advanced features, because (while useful) I don't think it's something that would be oft-used (@fyurie can you confirm?)

**Comment by Fyurie** [comment]
Yeah, it only really applies to Akagami right now, and spot-episodes for BokuMachi coz >duplex.

It's probably worth having but it's not significantly mandatory.
